src/strategies/gmc/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'info-gokumarket';
+export const version = '0.0.1';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'balanceOf', [address])
```

Comment:
Hi @info-gokumarket you can use `erc20-balance-of` strategy for this

Reply:
Hi
Yes I have used the erc20 balance strategy only and the test are also passing from my side
Could you please check again

Reply:
No, i mean instead of creating a new strategy you can use `erc20-balance-of` strategy in your space settings and get the balance

Reply:
Yes Thats fine
But why its failing on your side? Its not failing on my side

Reply:
its showing strategy not found in your logs

Reply:
![image](https://user-images.githubusercontent.com/15967809/161414626-640f7e90-c5a0-43c8-94a2-8a30fe8f4cab.png)
I mean like this @info-gokumarket you can use `erc20-balance-of` in your space settings, we don't need duplicate strategy

Reply:
Okay!